pkgs/applications/video/pitivi/default.nix

```diff
@@ -46,6 +46,8 @@ in stdenv.mkDerivation rec {
     dbus-python
   ]);
 
+  PYTHONPATH = "${python3Packages.gst-python}/lib/${python3Packages.python.sitePackages}";
```

Comment:
I first wondered what this change fixed, but saw that `CONFIGURED_PYTHONPATH` changed within the wrapped python script pick it up, which is part of the meson buildsystem: https://github.com/GNOME/pitivi/blob/master/meson.build#L28

Reply:
Honestly I didn't know either but when I was searching around I saw somewhere that this error was related to Python not finding `gst-python` and then I dug around a bit and found this little trick.

Reply:
`gst-python` has a `lib` inside a `lib` folder? Shouldn't that be fixed instead...?

Reply:
@FRidh Yeah, it does. I've been looking a bit at that as well and hope to fix it. My first fix attempt didn't work so I did Pitivi instead since that's what I wanted in the first place :-)

I'll continue looking at gst-python…

Reply:
@rycee the `preConfigure` is incorrect:
```
$out/lib/${python.sitePackages}
```
should be
```
$out/${python.sitePackages}
```

Reply:
I swear that was the "first fix attempt" I mentioned and it didn't work. But now I tried it again and it worked!

💩

I'll make a PR for gst-python with the corrected output path…